charts/athens-proxy/templates/_helpers.tpl

```diff
@@ -3,8 +3,8 @@
 {{- printf "%s-%s" .Release.Name $name | trunc 63 | trimSuffix "-" -}}
 {{- end -}}
 {{- define "livenessPath" -}}
-{{- if eq .Values.image.tag "v0.3.0" -}}/{{- else -}}/healthz{{- end -}}
+{{- if contains "v0.2" .Values.image.tag -}}/healthz{{- else -}}/{{- end -}}
```

Comment:
Maybe it's `v0.3` instead of `v0.2`?

Reply:
by my understanding v0.3.x` has different readiness/liveness check endpoints just `/`,
so the new logic is fine :)